src/parseexpr.jl

```diff
@@ -4,7 +4,11 @@
 #  file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 function tryParseIdxSet(arg::Expr)
-    if arg.head === :(=)
+    if arg.head === :kw  # Julia 1.0 onwards.
```

Comment:
It's better to use explicit VERSION checks so that they can be removed semi-automatically in the future.

Reply:
Should the version check be added for the old behaviour of the new behaviour?

Reply:
I don't think we have a convention, just as long as it's clear which code blocks can be removed when.